readme.md

```diff
@@ -30,16 +30,27 @@ The output is a JSON object in the form:
 
 ```json
 {
-    "bundle_name": {
-        "asset_kind": "/public/path/to/asset"
-    }
+    "chunks": {
+      "bundle_name": {
+          "asset_kind": "/public/path/to/asset"
+      }
+    },
+    "images": [{
+        "name": "hello.svg",
+        "path": "/public/path/to/asset/hello.svg"
+      }
+    ]
 }
 ```
 
 Where:
 
-  * `"bundle_name"` is the name of the bundle (the key of the entry object in your webpack config, or "main" if your entry is an array).
-  * `"asset_kind"` is the camel-cased file extension of the asset
+  * `chunks` is an object that contains all entry points listed in your webpack config
```

Comment:
@kossnocorp Sure, I'll update PR to use `entries` everywhere instead of `chunks`.